miniserver/routes/index.js

```diff
@@ -0,0 +1,141 @@
+const express = require('express')
+const config = require('../config')
+const router = express.Router()
+const { StatusCodes } = require('http-status-codes')
+const { Logger } = require('../logger')
+const NodeCache = require('node-cache')
+const Cache = new NodeCache()
+const { phone } = require('phone')
+const Twilio = require('twilio')
+const utils = require('../utils')
+const { User } = require('../src/data/user')
+const { isEqual, pick } = require('lodash')
+const stringify = require('json-stable-stringify')
+const { Setting } = require('../src/data/setting')
+const { Request } = require('../src/data/request')
+const blockchain = require('../blockchain')
+const { ethers } = require('ethers')
+
+/**
+* @openapi
+* /health:
+*   get:
+*     description: Checks the health of the miniwallet-server
+*     responses:
+*       200:
+*         description: OK indicates server is running
+*/
+router.get('/health', async (req, res) => {
+  Logger.log('[/health]', req.fingerprint)
+  res.send('OK').end()
+})
+
+const checkfromTwilio = async (req, res, next) => {
+  // check valid Twilio Request
+  if (!Twilio.validateRequest(config.twilio.token, req.headers['x-twilio-signature'], ('https://' + req.get('host') + '/sms'), req.body)) {
+    return res.status(StatusCodes.BAD_REQUEST).json({ error: 'request not from twilio' })
+  }
+  next()
+}
+
+function isNumeric (n) {
+  return !isNaN(parseFloat(n)) && isFinite(n)
+}
+
+const parseSMS = async (req, res, next) => {
+  const message = {}
+  const { From: senderPhoneNumber, Body: smsBody } = req.body
+  const requestor = await User.findByPhone({ phone: senderPhoneNumber })
+  message.requestor = requestor
+  const smsParams = smsBody.split(/(\s+)/).filter(e => { return e.trim().length > 0 })
+  //   for (let i = 0; i < smsParams.length; i += 1) {
+  //     Logger.log(`smsParams[${i}]: ${smsParams[i]}`)
+  //     Logger.log(`smsParams.length: ${smsParams.length}`)
+  //   }
+  if (smsParams.length < 1) {
+    return res.status(StatusCodes.BAD_REQUEST).json({ error: 'empty sms command' })
+  }
+  let funder = {}
```

Comment:
use plain variables whenever possible - it is unclear here what funder may contain

Reply:
fixed 🙏